### PR TITLE
Feature/warccims 70 improve errors endpoint for hgms

### PR DESCRIPTION
**Pull request by jdarcey-usgs**
## 1.3.0 - 09/24/2019
 * Update the getPayloadDecoder method so that the type name is included

### CODE REVIEW

File: gov.usgs.warc.iridium.sbd.decoder/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 	<groupId>gov.usgs.warc</groupId>
```
Comment (mckelvym-usgs) [inline gov.usgs.warc.iridium.sbd.decoder/pom.xml:6]: Can you update:
 * guava to 28.1-jre, 
 * jdt.annotation to 2.2.300
 * spring boot to 2.1.8.RELEASE

Comment (jdarcey-usgs) [reply gov.usgs.warc.iridium.sbd.decoder/pom.xml:6]: np

**Comment by mckelvym-usgs** [comment]
Please also update code.json (version & last updated)

**Review by mckelvym-usgs — APPROVED** [review]
Approved. Please merge and tag.

**Pull request merged**